docs/metrics/exemplars/README.md

```diff
@@ -1,99 +1,168 @@
 # Using Exemplars in OpenTelemetry .NET
 
-Exemplars are example data points for aggregated data. They provide specific
-context to otherwise general aggregations. One common use case is to gain
-ability to correlate metrics to traces (and logs). While OpenTelemetry .NET
-supports Exemplars, it is only useful if the telemetry backend also supports the
-capabilities. This tutorial uses well known open source backends to demonstrate
-the concept. The following are the components involved:
+<!-- markdownlint-disable MD033 -->
```

Comment:
Did you copy this from somewhere?

Asking because repo _should_ be configured to allow `details` \ `summary`: https://github.com/open-telemetry/opentelemetry-dotnet/blob/6c0327c4c6e75ac77566688e663d31a22f6efaa3/.markdownlint.yaml#L9-L12

Curious if you got a warning or you were being preemptive.

Reply:
Copied from https://github.com/open-telemetry/opentelemetry-dotnet/blob/6c0327c4c6e75ac77566688e663d31a22f6efaa3/docs/metrics/README.md?plain=1#L3.

Reply:
#5638